apps/worker/src/batch_announcer/batch.announcer.service.ts

```diff
@@ -47,7 +45,9 @@ export class BatchAnnouncementService extends WorkerHost implements OnApplicatio
   async process(job: Job<IBatchAnnouncerJobData, any, string>): Promise<any> {
```

Comment:
I think `IBatchAnnouncerJobData` contract is not that accurate. So right now we are reusing `IRequestJob` inside that for each announcement but in reality that should be an announcement type like https://github.com/AmplicaLabs/dsnp-gateway-prototype/blob/main/services/dsnp.ts#L17 

so flow is like the following
inside RequestProcessor:
1- check uploaded assets existence
2- create activity content type from request just like https://github.com/AmplicaLabs/dsnp-gateway-prototype/blob/main/handlers/content.ts#L131
3- upload/pin that activity content in ipfs
4- create announcement type like https://github.com/AmplicaLabs/dsnp-gateway-prototype/blob/main/handlers/content.ts#L135-L137
5- enqueue created announcement to it's correct queue based on announcement type

inside batcher:
we will batch those single announcements to a batched type with array of announcement and some metadata about the batch

ex:
```typescript
BatchedAnnouncements {
     referenceId: string;
     batchId: string;
     schemaId: number;
     announcements: Announcement[]
}
```

inside batch announcer:
1- use that batch type to create batch parquet file
2- pin the parquet file in ipfs
3-  enqueue a message for publisher

Reply:
I fixed this one